src/Airport.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+
+namespace AirPort
+{
+    public class TrafficController
```

Comment:
what is the difference between TrafficController and AirMap?  Looks like the same thing.  I would get rid of one of them.

Reply:
yeah, it was a glitch, it's an extra file.